test/connectors/test_torch_connector.py

```diff
@@ -61,6 +62,42 @@ def setUp(self):
 
         torch.manual_seed(algorithm_globals.random_seed)
 
+    @abstractmethod
```

Comment:
If they need to derive from `TestTorchConnector` yes but that is the same solution use in Nature. Another solution could be adding a `pylint: ignore` to ignore warnings on not found methods but that would cause any method misspelled in this file to fail only on runtime.
https://github.com/Qiskit/qiskit-nature/blob/bff9c6a5fe674485c8588add27534bcad43b3276/test/drivers/second_quantization/test_driver.py#L46

Reply:
I guess its been a while since those driver tests and I did not remember them being done like this - I recalled they had the common parent with the tests just not that it included signatures like these too. Maybe there is no better way to do this type of thing to avoid this "awkwardness". It all works so I guess its good.

Reply:
I refactored it to a common base class since they were repeated in two base classes.